### PR TITLE
link to code for memory estimate

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

**Comment by kkraune** [comment]
@geirst

File: en/streaming-search.html [hunk 1]
```diff
@@ -181,16 +181,10 @@ <h2 id="memory-sizing">Memory sizing</h2>
...
-<pre>
-&lt;tuning&gt;
-  &lt;searchnode&gt;
-    &lt;resizing&gt;
-      &lt;initialdocumentcount&gt;5000000&lt;/initialdocumentcount&gt;
-</pre>
-Note: This is a hard limit if the node does not have memory to keep more than one attribute in memory!
+</p>
+<p>
+See <a href="https://github.com/vespa-engine/vespa/blob/3b62cc5a2e7393c9399e7e8bda05085a91fc2ecc/config-model/src/main/java/com/yahoo/vespa/model/search/NodeResourcesTuning.java#L20">
```
Comment (geirst) [inline en/streaming-search.html:186]: I am not sure we should link to this at all. It is very low-level to make any sense for the user.

**Review by geirst — APPROVED** [review]
👍

File: en/streaming-search.html (hunk 1 above)
Comment (kkraune) [reply en/streaming-search.html:186]: right. the question comes up now and then for such apps - how many docs can we estimate per node. Maybe we should just say "estimate 50 bytes per doc"?

Comment (geirst) [reply en/streaming-search.html:186]: Yes, I think that is better.

**Pull request merged**